pkg/controller/common/operator/flags.go

```diff
@@ -10,8 +10,10 @@ const (
 	CACertValidityFlag           = "ca-cert-validity"
 	CertRotateBeforeFlag         = "cert-rotate-before"
 	CertValidityFlag             = "cert-validity"
+	ConfigFlag                   = "config"
 	ContainerRegistryFlag        = "container-registry"
 	DebugHTTPListenFlag          = "debug-http-listen"
+	DisableConfigWatch           = "disable-config-watch"
```

Comment:
Thanks for pointing out the inconsistency. I don't think expanding `config` to `configuration` achieves much (there's no ambiguity) other than making the flags comically long. Since the webhook flag hasn't been released yet, I think I will rename that instead. Even Kubernetes uses `ConfigMap` instead of `ConfigurationMap` :smile:.